organizers/tables.py

```diff
@@ -72,7 +72,7 @@ class Meta:
         template = 'django_tables2/bootstrap-responsive.html'
         fields = ['counter', 'email', 'vote_count', 'skip_count', 'total_count']
         empty_text = 'No organizers voted yet... Why? :\'('
-        order_by = '-total_count'
+        ordering = ['-total_count', '-vote_count', '-skip_count']
```

Comment:
Why the change from `order_by` to `ordering`? Does it work as expected?
 
It may be fine, I have no idea. But I haven't found any reference to `ordering` in the `django-tables2` documentation. [It only talks about](https://django-tables2.readthedocs.io/en/latest/pages/api-reference.html?#table) `order_by`.

Reply:
If this was intended and it works there's no need to change it back, of course. 
But maybe it's a mix-up with [Django's ModelAdmin.ordering](https://docs.djangoproject.com/en/2.1/ref/contrib/admin/#django.contrib.admin.ModelAdmin.ordering).